### PR TITLE
Feat: Add publishing date for non-official sources

### PR DESCRIPTION
**Pull request by KievDevel**
Motivation: We use sha sum to calculate if build is updated, but sometimes I'm missing timestamps when same version is released over and over again, ending up testing wrong version of the build.

### CODE REVIEW

File: ipc/apps.ts [hunk 1]
```diff
@@ -34,6 +34,7 @@ interface Installed {
     repositoryUrl?: UrlString;
     html?: string;
     installed: {
+        publishTimestamp?: string;
```
Comment (datenreisender) [inline ipc/apps.ts:37]: Whether you also need to add this, depends on what you want to do with the timestamp (which I do not know).

File: scripts/nordic-publish.ts [hunk 2]
```diff
@@ -269,6 +270,7 @@ const updateLegacyAppInfo = (appInfo: LegacyAppInfo, app: App) => {
             ...appInfo.versions,
             [app.version]: {
                 dist: {
+                    publishTimestamp: new Date().toISOString(),
```
Comment (datenreisender) [inline scripts/nordic-publish.ts:273]: This isn't really needed but also causes no harm.

File: ipc/apps.ts (hunk 1 above)
Comment (KievDevel) [reply ipc/apps.ts:37]: Will revisit this. Initially I want to display timestamp of the already installed version of the app. Seems like the right place to add.

Comment (datenreisender) [reply ipc/apps.ts:37]: 👍

**Review by datenreisender — APPROVED** [review]

File: Changelog.md [hunk 3]
```diff
@@ -7,6 +7,12 @@ This project does _not_ adhere to
 [Semantic Versioning](https://semver.org/spec/v2.0.0.html) but contrary to it
 every new version is a new major version.
 
+## 175.0.0 - UNRELEASED
+
+### Added
+
+-   Publish timestamp for nordic-publish for non-official sources.
+
```
Comment (boundlesscalm) [inline Changelog.md:15]: Move into v174

**Review by datenreisender — APPROVED** [review]

**Pull request merged**